angularjs/angular.d.ts

```diff
@@ -1259,6 +1229,15 @@ declare module angular {
     }
 
     /**
+     * xhrFactory
+     * Replace or decorate this service to create your own custom XMLHttpRequest objects.
+     * see https://docs.angularjs.org/api/ng/service/$xhrFactory
+     */
+    interface IXhrFactory {
+        <T>(method: string, url: string): T;
```

Comment:
put the generic on the interface, `interface IXhrFactor`, to be consistent with the rest of these typings